Module2/springhometask/src/main/java/com/epam/spring/dao/impl/StaticEventDao.java

```diff
@@ -16,6 +16,10 @@
     @Autowired
```

Comment:
class StaticEventDao not implementing related interface - StaticEventDao 
StaticEventDao - is not good name, it should not contain "static" word

Reply:
renamed